app/views/mailer/mfa_recommendation_announcement.text.erb

```diff
@@ -0,0 +1,15 @@
+<% @title = "Enable MFA on your RubyGems account" %>
+<% @sub_title = "👋 Hi #{@user.handle}" %>
+
+Hi there,
+
+Today, we've announced our security-focused ambitions to the community. Attacks on the software supply chain are increasing and the second most common attack is account takeovers.
+
+We are officially recommending maintainers of packages with more than 165 million downloads to enable MFA. On August 15, 2022, we will begin requiring maintainers of packages with more than 180 million downloads to have MFA enabled. 
+
+While this policy impacts specific RubyGems users, everyone can do their part to help combat account takeovers. Please enable multi-factor authentication on your RubyGems account 🙏. We recommend setting the MFA level to `UI and API`. However, `UI and gem signin` is acceptable too (https://guides.rubygems.org/setting-up-multifactor-authentication/#authentication-levels).
+
+For more information, check out our announcement (#link-to-blog-post-tbd).
```

Comment:
Note: I haven't added the actual link to the blog post since the URL doesn't exist. Although, in theory if everything goes according to plan, we do what the link _should_ be:

`https://blog.rubygems.org/2022/06/13/making-packages-more-secure.html`